pkgs/development/python-modules/zulip-emoji-mapping/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  python3Packages,
+}:
+
+buildPythonPackage rec {
+  pname = "zulip-emoji-mapping";
+  version = "1.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "GearKite";
+    repo = "zulip-emoji-mapping";
+    rev = "973afff3466b8423298c3e671bf3011e0815626b";
+    hash = "sha256-logm5uAnLAcFqI7mUxKEO9ZmHqRkd6CFiCW4B5tqZzg=";
+  };
+
+  dependencies = with python3Packages; [
```

```suggestion
  buildInputs = with python3Packages; [
```
`setuptools` is only needed for building the package.